examples/bits.rs

```diff
@@ -9,8 +9,10 @@
 //! [@myrrlyn]: //github.com/myrrlyn
 //! [`bitvec`]: //crates.io/crates/bitvec
 
-// The `bitvec` crate provides a bunch of types for us to use with one import
+// The `bitvec` crate provides a lot of types; we only need a few:
 use bitvec::prelude::*;
+    // This trait provides the `.bits::<_>()` method which we use on Rust types
```

Comment:
I think this comment may have been placed here by accident? Did you mean to update it?

Reply:
`git add -P` is my friend, until it isn't,